### ISSUE TITLE
Dev Seeds

### ISSUE THREAD

**Issue by voodoorai2000** [consul/consul#1780]
What
====
- Setup comprehensive dev seed data

Why
===
- [Dev seeds](https://github.com/consul/consul/blob/master/db/dev_seeds.rb) have grown and it's hard to follow
- We are using random associations which produce unexpected seed data, not covering the main phases for each module

How
===
- Split up dev_seeds.rb into smaller logical files (settings, users, budgets, legistions, etc)
- Cover correctly the different types of users and phases for the main modules (users, budgets, legislations, proposals)

**Comment by aitbw** [comment]
I'm on it.

**Comment by bertocq** [comment]
Hey @aitbw I'm making some improvements for this tasks, not sure if you've worked already on it but if so please share a branch or work on top of this changes:

https://github.com/consul/consul/commit/54e6c5fc5ce04d2a391afe59b36c2c6abd02e06a
https://github.com/consul/consul/commit/500db4b5d87883e4b758a7ce721dfac7af8f04be

The idea is to:
- Use individual files for each section to make it easier to read, and also resolve merge conflicts
- Use yml files (seeds.yml) to gather translatable stuff so a Demo can be created on different languages... as well as separating content from code when we start adding 5-10 example Proposals or Poll Questions

This is not an issue that can be tackled at once, its better to go section by section creating some example data... or even better for each new feature we add, or each time we modify one.. remembering to change/update/upgrade dev_seeds 👌
